### PR TITLE
Add baseline model

### PR DESCRIPTION
**Pull request by jeswan**
The baseline model is a global disparity shift where we take the left image and shift it in the range of [-192, 192]. We calculate the lowest MAE and global disparity on the validation set (same as Deep3D) for these shifts and output the results to a file. This does not yet support evaluation on a test set. Baseline dataloaders are similar to the other datasets so we can use the same data splits here.

Run using:
`python src/models/baseline_model.py --data_dir ~/nyu_ms_ds/capstone/daVinci `

### CODE REVIEW

File: src/models/baseline_model.py [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+            for batch_idx, batch in enumerate(tqdm(dataloader)):
+                imgs, targets = batch
+                for img, target in zip(imgs, targets):
+                    img = img.numpy()
+                    target = target.numpy()
+                    img = img.transpose(1, 2, 0)
+                    target = target.transpose(1, 2, 0)
+
+                    func = partial(calc_mae_translation, img, target, mae_list)
+                    p.map(func, enumerate(translations))
```
Comment (jeswan) [inline src/models/baseline_model.py:84]: parallelization

File: src/models/baseline_model.py [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+                imgs, targets = batch
+                for img, target in zip(imgs, targets):
+                    img = img.numpy()
+                    target = target.numpy()
+                    img = img.transpose(1,2,0)
+                    target = target.transpose(1,2,0)
+
+                    func = partial(calc_mae_translation, img, target, mae_list)
+                    p.map(func, enumerate(translations))
+                break
```
Comment (jeswan) [inline src/models/baseline_model.py]:
```suggestion
```

File: setup.py [hunk 3]
```diff
@@ -10,6 +10,6 @@
     author_email='',
     # REPLACE WITH YOUR OWN GITHUB PROJECT LINK
     url='https://github.com/annikabrundyn/da_vinci',
-    install_requires=['pytorch-lightning'],
-    packages=find_packages(),
+    package_dir={"": "src"},
```
Comment (annikabrundyn) [inline setup.py:13]: been meaning to ask you if this changes anything abou the package install?
